BookingSystem/Frontend/src/hotelSearch/containers/HotelSearchContainer.js

```diff
@@ -28,36 +26,45 @@ class HotelSearchContainer extends Component {
   state = {
```

Comment:
Thinking if the filters should be a part of the store... It seems to be necessary there to have the complete state: filters, page option and list of hotels. What do you think?